### DISCUSSION TITLE
win11 版本 V1.39.X 无法打开问题

### DISCUSSION THREAD

**Discussion by spiketang** [electerm/electerm#3421] [Q&A]
安装V1.39.X的便捷版还是安装版本，都无法使用，打开就卡死了。最终退到1.38.86才能正常使用；
![image](https://github.com/electerm/electerm/assets/3364115/3a344910-9421-465b-851e-8912f2b26440)

**Comment by zxdong262** [comment]
可以命令行运行1.39.5看看什么错误吗

**Reply by spiketang** [reply]
我刚刚试了一下重新1.39.5，不过不是覆盖升级，是单独安装，又能正常打开了。之前是安装升级后无法打开。我删除掉再试一下升级的功能。
![image](https://github.com/electerm/electerm/assets/3364115/eab583e4-a211-4f99-b532-9d90d9bc2ca1)

**Reply by spiketang** [reply]
测试了从1.38到1.39升级无问题，估计就两种可能：
1、我之前的版本太低了，升级过程中，可能有其它异常情况，导致升级后，打开electerm就卡死在开始界面；
2、win11系统自身问题，可能是内存，可能是其它方面异常，估计需要重启动电脑或清理一下内存垃圾就OK
